config/default.yml

```diff
@@ -2058,6 +2058,9 @@ Lint/UnreachableLoop:
   Description: 'This cop checks for loops that will have at most one iteration.'
   Enabled: true
   VersionAdded: '0.89'
+  VersionChanged: <<next>>
+  IgnoredPatterns:
```

Comment:
I'd add a comment explaining what the default configuration targets, as it's not immediately obviously, especially if you're not using RSpec.

Reply:
Added a comment, let me know if you think it needs more info.

Reply:
I think you may have forgotten to push.

Reply:
Oops! Sorry, pushed now.

Reply:
@bbatsov any changes necessary do you think?

Reply:
Nope. I think we're good here.